lib/controllers/AssetController.ts

```diff
@@ -46,17 +58,35 @@ export class AssetController extends CRUDController {
     const model = request.getBodyString('model');
     const reference = request.getBodyString('reference');
 
-    if (! request.input.resource._id) {
+    if (! request.input.args._id) {
```

Comment:
You should use `request.getId()`

Reply:
Here i believe it's not worth since request.getId() throw an error if it's not defined we end up with a try/catch to generate it